superweather/superweather/LocationTool.h

```diff
@@ -0,0 +1,21 @@
+//
+//  LocationTool.h
+//  superweather
+//
+//  Created by mikael petersson on 2016-04-17.
+//  Copyright © 2016 mikael petersson. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+@interface LocationTool : NSObject<CLLocationManagerDelegate>
+
+@property (strong, nonatomic) CLLocationManager *locationManager;
+
+
+-(LocationTool*)init;
```

Comment:
You don't need to that is the method signature that it inherits from NSObject, see: https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSObject_Class/#//apple_ref/occ/instm/NSObject/init